node/hybrid-node-tests/Common-test/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "common-test",
+    "version": "1.0.0",
+    "description": "",
+    "main": "Common-test.cjs",
+    "type": "commonjs",
+    "scripts": {
+        "build-before-test": "cd ../../../node && npm run build",
+        "test-package-for-common": "npm run build-before-test && node Common-test.cjs",
+        "test-for-ci": "npm run test-package-for-common",
+        "prettier:check:ci": "./node_modules/.bin/prettier --check .",
+        "prettier:format": "./node_modules/.bin/prettier --write . "
+    },
+    "author": "",
```

Comment:
please fix the authors throughout to be `Amazon Web Services`, and the licenses to be `Apache-2.0`